src/avs/AVSDetails.jsx

```diff
@@ -1,330 +1,164 @@
-      liquidityStakedTVL +
-      Number(eigenTVL / BigInt(1e18));
-
-    const totalEthDistributionData = [
-      beaconEntry && {
-        ...beaconEntry,
-        token: beaconEntry.token,
-        tvl: convertToEther(beaconTVL),
-        tvlPercentage: ((convertToEther(beaconTVL) / totalTVL) * 100).toFixed(2)
-      },
-      eigenEntry && {
-        ...eigenEntry,
-        token: eigenEntry.token,
-        tvl: convertToEther(eigenTVL),
-        tvlPercentage: ((convertToEther(eigenTVL) / totalTVL) * 100).toFixed(2)
-      },
-      {
-        name: 'Liquidity Staked Tokens',
-        logo: 'https://w7.pngwing.com/pngs/268/1013/png-transparent-ethereum-eth-hd-logo-thumbnail.png',
-        tvl: liquidityStakedTVL,
-        tvlPercentage: ((liquidityStakedTVL / totalTVL) * 100).toFixed(2)
-      }
-    ].filter(Boolean);
-
-    return {
-      lstDistributionData,
-      eigenTVL,
-      beaconTVL,
-      liquidityStakedTVL,
-      totalTVL,
-      totalEthDistributionData
-    };
-  }
-
-  const convertToEther = useMemo(
-    () => bigIntValue => Number(bigIntValue / BigInt(1e18)),
-    []
-  );
+    ethRate: 1
+  });
+  const { avsService } = useServices();
+  const compact = !useTailwindBreakpoint('lg');
 
   useEffect(() => {
-    async function fetchAvsDetails() {
-      try {
-        const avsDetailsData = await avsService.getAVSDetails(avsAddress);
-        const strategiesMap = createStrategiesMap(
-          avsDetailsData.strategies || []
-        );
-        const derivedValues = calculateDerivedValues(
-          avsDetailsData,
-          strategiesMap
-        );
-
-        dispatch({
-          avsDetails: avsDetailsData,
-          strategiesMap,
-          ...derivedValues,
-          rate: avsDetailsData.rate
-        });
-      } catch (error) {
-        // TODO: handle error
+    dispatch({ isAVSLoading: true });
+    (async () => {
+      const avs = await avsService.getAVSDetails(address);
+
+      const strategies = avs.strategies.reduce((acc, strategy) => {
+        acc[strategy.address] = BigInt(strategy.tokens) / BigInt(1e18);
+        return acc;
+      }, Object.create(null));
+
+      const totals = { lst: 0n, eth: 0n, eigen: 0n };
+
+      for (const strategy in strategies) {
+        if (strategy === EIGEN_STRATEGY) {
+          totals.eigen += strategies[strategy];
+        } else if (strategy === BEACON_STRATEGY) {
+          totals.eth += strategies[strategy];
+        } else {
+          totals.lst += strategies[strategy];
+        }
       }
-    }
-    fetchAvsDetails();
-  }, [avsService, dispatch, avsAddress]);
 
-  if (state.avsDetails === null) {
-    return null;
-  }
+      totals.eigen = Number(totals.eigen);
+      totals.eth = Number(totals.eth);
+      totals.lst = Number(totals.lst);
+
+      dispatch({
+        avs,
+        ethRate: avs.rate,
+        strategies,
+        totalTokens: totals,
+        isAVSLoading: false
+      });
+    })();
+  }, []);
```

Comment:
You forgot the dependencies:

```suggestion
  }, [address, avsService, dispatch]);
```